### ISSUE TITLE
[Crash] Crash when adding songs to a playlist in Search

### ISSUE THREAD

**Issue by dossma** [chr56/Phonograph_Plus#232]
### Bug Summary

```
Crash Report:

App version:     1.8.4 (1084)
Git Commit Hash: ff9861b5a23611e2a02b2b62eebc0f7dbd119bca
Package name:    player.phonograph.plus
Release favor:   modernStable
Android version: 14 (API 34)
Architecture:    arm64-v8a
Soc:             Tensor G3/shiba (by Google) 
Device brand:    google (by Google)
Device model:    shiba/Pixel 8 (code shiba)
Build version:   2024100800
                 (2024100800)
Language:        de
Memory:          1537/7354 
Screen:          2205x1080 (dpi 420)
Permissions:     Storage(READ WRITE)

Stacktrace:
java.lang.IllegalArgumentException: Failed requirement.
	at nb.i.b(SourceFile:53)
	at cc.l.J(SourceFile:71)
	at nf.a.invokeSuspend(SourceFile:250)
	at r9.a.resumeWith(SourceFile:9)
	at ka.f0.run(SourceFile:102)
	at androidx.recyclerview.widget.u.run(SourceFile:188)
	at ra.j.run(SourceFile:3)
	at ra.a.run(SourceFile:93)
	Suppressed: pa.e: [m1{Cancelling}@15f4fb1, Dispatchers.Main.immediate]
```

### Version

1.8.4

### Steps to reproduce the bug

I had just stopped a very long playlist from playing. Then I searched for a song to add it to another playlist.

**Comment by chr56** [comment]
Fixed in b98e3001. Use this [Preview Version](https://github.com/user-attachments/files/17353089/Preview-SafCrash.zip) for checking and testing.

**Comment by chr56** [comment]
[v1.8.5](https://github.com/chr56/Phonograph_Plus/releases/tag/v1.8.5) has been released! 

This issue _should_ be fixed.

**Close**! 
Reopen the issue if it happens again, or create a new issue for a similar one or a newly introduced one.
